### ISSUE TITLE
Commands randomly fail because "Exec command ... is already running"

### ISSUE THREAD

**Issue by aramperes** [nektos/act#172]
I will randomly get errors that say:

```
[Deploy front-end/deploy] ⭐  Run jq -r '.version' projects/core/package.json > /tmp/version
| Error: Exec command a0d637d4b9b794f61bbb78d3a83337119ce98b9540840e1acc0cda150ec1bdd6 is already running
[Deploy front-end/deploy]   ✅  Success - jq -r '.version' projects/core/package.json > /tmp/version
```

and the step is just marked as "Success", so the build continues.

I'm building a Node-based workflow in the `ubuntu-latest` image, however the error seems unrelated to Node (it happens with other commands such as jq).

I made sure that no act container is already running before starting it, and pruned all stopped containers/volumes as well.

Relevant system information:
```
act version 0.2.7
Docker version 19.03.8, build afacb8b7f0
Ubuntu 19.10
```

**Comment by cplee** [comment]
Looks like a duplicate of #165
